modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/PageRenderer/PageRenderer.es.js

```diff
@@ -217,6 +219,26 @@ class PageRenderer extends Component {
 		this.emit('fieldFocused', event);
 	}
 
+	_handleMovePageDownClicked() {
+		const {pageIndex} = this;
+		const {dispatch} = this.context;
+
+		dispatch('pagesSwapped', {
+			firstIndex: pageIndex,
+			secondIndex: pageIndex + 1,
+		});
+	}
+
+	_handleMovePageUpClicked() {
+		const {pageIndex} = this;
+		const {dispatch} = this.context;
+
+		dispatch('pagesSwapped', {
+			firstIndex: pageIndex,
+			secondIndex: pageIndex - 1,
```

Comment:
I haven't tested it and probably the UI already protects against it, but I'm asking just in case: Can this potentially evaluate for a negative number and break things? For example if pageIndex is `0`?

Reply:
No, in this case pageIndex is never 0. The first page (0) can only can only be moved down. The move up button for page 0 is disabled.